pyproject.toml

```diff
@@ -703,3 +700,6 @@ split-on-trailing-comma = false
 
 [tool.ruff.mccabe]
 max-complexity = 25
+
+[tool.ruff.format]
+exclude = ["/generated/"]
```

Comment:
It may be necessary to change this to `/generated/**` See https://github.com/astral-sh/ruff/issues/8267

Reply:
In fact, I think this is unnecessary altogether, since the generated files do get filtered through `black` at generation time. I suspect it's just here because there hasn't been a reason to spend cycles on Black re-checking them... whereas with Ruff, well...